recipes/formencode/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "FormEncode" %}
+{% set version = "1.3.1" %}
+{% set hash_value = "ada2f51792b1b484e5bb7b6cc14acfc1bc11fafc967cf015cd57e856053ca7f6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - formencode
+
+about:
+  home: http://formencode.org
+  license: MIT
```

Comment:
Can you manually package the license:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and include the following issue link as a comment in the reipe:
https://github.com/formencode/formencode/issues/91